__tests__/app.test.js

```diff
@@ -166,3 +166,107 @@ describe("GET /api/reviews/:review_id/comments", () => {
 			});
 	});
 });
+
+describe("PATCH /api/reviews/:review_id", () => {
+	test("status:200, returns the updated review object when told to increment", () => {
+		const newVotes = {
+			inc_votes: 3
+		};
+		return request(app)
+			.patch("/api/reviews/1")
+			.send(newVotes)
+			.expect(200)
+			.then(({ body }) => {
+				const { review } = body;
+				expect(review).toMatchObject({
+					owner: "mallionaire",
+					title: "Agricola",
+					designer: "Uwe Rosenberg",
+					review_id: 1,
+					category: "euro game",
+					review_img_url: "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+					created_at: "2021-01-18T10:00:20.514Z",
+					votes: 4
+				});
+			});
+	});
+
+	test("status:200, returns the updated review object when told to decrement", () => {
+		const newVotes = {
+			inc_votes: -1
+		};
+		return request(app)
+			.patch("/api/reviews/1")
+			.send(newVotes)
+			.expect(200)
+			.then(({ body }) => {
+				const { review } = body;
+				expect(review).toMatchObject({
+					owner: "mallionaire",
+					title: "Agricola",
+					designer: "Uwe Rosenberg",
+					review_id: 1,
+					category: "euro game",
+					review_img_url: "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+					created_at: "2021-01-18T10:00:20.514Z",
+					votes: 0
+				});
+			});
+	});
+
+	test("status:204, returns nothing when not told to increment or decrement", () => {
+		const newVotes = {
+			inc_votes: 0
+		};
+		return request(app)
+			.patch("/api/reviews/1")
+			.send(newVotes)
+			.expect(204)
+			.then(({ body }) => {
+				const { review } = body;
+				expect(review).toBeUndefined();
+			});
+	});
+
+	test("status:404, returns an appropriate error message when given review doesn't exist", () => {
+		const newVotes = {
+			inc_votes: 5
+		};
+		return request(app)
+			.patch("/api/reviews/9999")
+			.send(newVotes)
+			.expect(404)
+			.then(({ body }) => {
+				const { msg } = body;
+				expect(msg).toBe("Content not found");
+			});
+	});
+
+	test("status:400, returns an appropriate error message when sending a malformed body", () => {
+		const newVotes = {
+			someKeyThatIsntIncVotes: 100
+		};
+		return request(app)
+			.patch("/api/reviews/1")
+			.send(newVotes)
+			.expect(400)
+			.then(({ body }) => {
+				const { msg } = body;
+				expect(msg).toBe("Bad request");
+			});
+	});
+
+	test("status:400, returns an appropriate error message when sent body fails schema validation", () => {
+		const newVotes = {
+			inc_votes: "kitten"
+		};
+		return request(app)
+			.patch("/api/reviews/1")
+			.send(newVotes)
+			.expect(400)
+			.then(({ body }) => {
+				const { msg } = body;
+				expect(msg).toBe("Bad request");
+			});
+	});

```

Comment:
these tests are good, but there are a case that you might be missing:
 - what if I try and patch an invalid review id?